### PR TITLE
fix: Broken breadcrumbs when using custom BASE_URL

### PR DESCRIPTION
**Pull request by ddl-rliu**
# TL;DR
Typo – causes breadcrumbs to not appear in flyteconsole. To fix, use BASE_URL not BASE_PATH.

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description

Missing breadcrumbs for custom BASE_URL like `/custom-base/console`

image

Expectation: breadcrumbs are present for custom BASE_URL Flyte instance.

image

This happens because when `BASE_PATH` is used instead of `BASE_URL`, the logic that generates the breadcrumbs at `breadcrumbRegistry.breadcrumbBuilder` ends up using the wrong `BASE_URL` value (this starts from `makeUrlSegments`). This causes it to parse the window location incorrectly, and the resulting generated breadcrumbs is missing the expected entries. 

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

File: packages/oss-console/src/components/Breadcrumbs/registry/index.ts [hunk 1]
```diff
@@ -106,7 +106,7 @@ export class BreadcrumbRegistry {
 
   static makeUrlSegments(location: Location, projectId = '', domainId = '') {
     const pathName = location.pathname;
-    const basePath = process.env.BASE_PATH || '/console';
```
Comment (ddl-rliu) [inline packages/oss-console/src/components/Breadcrumbs/registry/index.ts:109]: I believe BASE_PATH is a typo here, since the `process.env` object looks like
```
{
    NODE_ENV: "production",
    ADMIN_API: "//localhost:30080",
    ADMIN_API_URL: "localhost:30080",
    BASE_URL: "/console",
    BASE_HREF: "",
    DISABLE_CONSOLE_ROUTE_PREFIX: "",
    MAINTENANCE_MODE: ""
}
```
(in the case of a custom path, it looks like `... BASEURL: "/custom-path/console"`

And double checking the repo code, BASE_PATH has no other instance of usage, pointing to this being a potential typo.

**Comment by eapolinario** [comment]
@jsonporter , can you take a look?

**Review by eapolinario — APPROVED** [review]
